src/pages/Pool/index.tsx

```diff
@@ -89,6 +90,16 @@ const NoLiquidity = styled.div`
   max-width: 300px;
   min-height: 25vh;
 `
+
+const WrongNetwork = styled.div`
```

Comment:
this styling seems very related to the `NoLiquidity` above. Maybe you can rename that to `ErrorContainer` or something?